code/edgeconsent/src/androidTest/java/com/adobe/marketing/mobile/MonitorExtension.java

```diff
@@ -225,30 +225,6 @@ private void processSharedStateRequest(final Event event) {
 		MobileCore.dispatchResponseEvent(responseEvent, event, null);
```

Comment:
Needs to be updated to use the new api for dispatch. Not sure if this PR includes removing deprecated api usage, but adding this for visibility.

Reply:
Yes, will do that in the Functional PR.  Please ignore this file right now.